spec/spec_helper.rb

```diff
@@ -118,31 +118,17 @@ def set_hieradata(hieradata)
   end
 
   c.before(:each) do
-    @spec_global_env_temp = Dir.mktmpdir('simpspec')
-
     if defined?(environment)
       set_environment(environment)
-      FileUtils.mkdir_p(File.join(@spec_global_env_temp,environment.to_s))
     end
 
-    # ensure the user running these tests has an accessible environmentpath
-    Puppet[:environmentpath] = @spec_global_env_temp
-    Puppet[:user] = Etc.getpwuid(Process.uid).name
-    Puppet[:group] = Etc.getgrgid(Process.gid).name
-
     # sanitize hieradata
     if defined?(hieradata)
       set_hieradata(hieradata.gsub(':','_'))
     elsif defined?(class_name)
       set_hieradata(class_name.gsub(':','_'))
     end
   end
-
-  c.after(:each) do
-    # clean up the mocked environmentpath
-    FileUtils.rm_rf(@spec_global_env_temp)
-    @spec_global_env_temp = nil
-  end
```

Comment:
Query-curiosity: Why the removal of these sections from the skeleton?

Reply:
@sharkbruhaha That seems to remain in newer updates that I have. Restored.